### DISCUSSION TITLE
Word Level Timestamps and Segments Precision

### DISCUSSION THREAD

**Discussion by tonytorm** [ggml-org/whisper.cpp#1584] [General]
This is an old topic - the one regarding word-level timestamps-. A few months ago, it was still considered experimental. I'm wondering if anything has changed since then. Although the updates in the new versions don't seem to indicate any progress, I thought it was worth asking here. 

Also, the precision and the length of the transcribed segments vary a lot from chunk to chunk, for our work having a precise segmentation would be amazing, even if only on a 'sentence' level.

**Comment by bobqianic** [comment]
See #1485
